modern-careers-page/index.html

```diff
@@ -23,99 +36,79 @@
             <video autoplay muted playsinline diablepictureinpicture loop alt="webpage design evolves from wireframe sketch to low-fidelity to interactive full-color">
-                <source src="images/EssexCareersPage_OverviewVideo.mp4" type="video/mp4">
+                <source src="modern-careers-page/images/EssexCareersPage_OverviewVideo.mp4" type="video/mp4">
             </video>
         </div>
 
         <div class="text-layout">
             <h2>A Manufacturing Company with a Culture to Brag About</h2>
             <p>Essex Industries makes critical parts for aircraft across the globe—which means they need the best of the best working for them. In order to attract top talent, they needed a robust careers page that showcased their welcoming, family-based culture.</p>
-            <h3>Skills</h3>
-            <div class="skills-list">
-                <p class="skill">User Experience Design</p>
-                <p class="skill">User Interface Design</p>
-                <p class="skill">Web Development</p>
-                <p class="skill">Marketing Strategy</p>
-            </div>
         </div>
 
         <div class="text-layout">
-            <h2>Crafting a clear user experience</h2>
+            <h2><span class="article-number">1</span>Crafting a clear user experience</h2>
             <p>Essex’s previous careers page, while functional, lacked a clear information flow. I organized the already existing sections into an intuitive progression—sharing who Essex is as an employer (culture, values, mission) before showing page users how they might fit in the company (available roles, employee testimonials.)</p>
         </div>
-        <div class="full-width">
-            <img src="images/EssexCareersPage_Image1.png" alt="detailed sketch of webpage wireframe and flow">
+        <div class="article-image card">
+            <div class="card-background"></div>
+            <img src="https://noahscottbraun.com/modern-careers-page/images/EssexCareersPage_Image1.png" alt="detailed sketch of webpage wireframe and flow">
         </div>
 
         <div class="text-layout">
-            <h2>The Interface-Lift</h2>
+            <h2><span class="article-number">2</span>The Interface-Lift</h2>
             <p>Created to appeal to serious military contractors, Essex’s brand is bare bones. However, I pushed the interface design in a (slightly) more fun direction for this page to give prospective employees an engaging experience featuring colorful and smooth animations.</p>
         </div>
-        <div class="full-width">
-            <img src="images/EssexCareersPage_Image2.png" alt="cleanly-designed webpage layouts for culture, values, benefits, and testimonials">
+        <div class="article-image card">
+            <div class="card-background"></div>
+            <img src="https://noahscottbraun.com/modern-careers-page/images/EssexCareersPage_Image2.png" alt="cleanly-designed webpage layouts for culture, values, benefits, and testimonials">
```

Comment:
The src here should be "modern-careers-page/images/EssexCareersPage_Image2.png"
Remove the domain name.